tests/python/contrib/test_ethosn/test_conv2d_transpose.py

```diff
@@ -115,7 +116,7 @@ def _get_model(
     [
         ((1, 2, 2, 1), (2, 2), (1, 1), 1, False),
         ((1, 2, 2, 5), (2, 2), (3, 5), 4, False),
-        ((1, 7, 7, 4), (2, 2), (7, 9), 8, True),
+        ((1, 7, 7, 4), (2, 2), (7, 7), 8, True),
```

Comment:
Curious, does this come from a change in the driver stack itself? It wasn't mentioned here: https://github.com/ARM-software/ethos-n-driver-stack/blob/main/CHANGELOG.md#2211 and, if so, this is wrong: https://github.com/ARM-software/ethos-n-driver-stack/blob/main/SUPPORTED.md?plain=1#L133

Reply:
An issue has been raised with the NPU software to track this.